src/main/java/org/testeditor/fixture/host/HostDriverFixture.java

```diff
@@ -136,6 +142,68 @@ public void sendControlCommand(ControlCommand value) {
         connection.doCommand("ascii"); // just to see if typed in successfully.
     }
 
+    /**
+     * Reads a value on a given position range start(x,y) and end(x,y)
+     * 
+     * @param locator
+     *            The Locator exists of starting(row,column) and end(row,column)
+     *            in the form "2;12;5;16" which means the start position is at
+     *            row = 2, column = 12 up to stop position row = 5, column 16.
+     * @return the String value which is read at the given position.
+     */
+
+    /**
+     * Reads a value on a given position range start(x,y) and end(x,y)
+     * 
+     * @param elementLocator
+     *            The elementlocator exists of position definitions with a
+     *            starting(row,column) and end(row,column) position in the form
+     *            "2;12;5;16" which means the start position is at row = 2,
+     *            column = 12 up to stop position row = 5, column 16.
+     * @param locatorType
+     *            see {@link LocatorStrategy} this is the type how the
+     *            elementlocator is defined.
+     * @return the String value which is read at the given position.
+     */
+    public String readValueAt(String elementLocator, LocatorStrategy locatorType) {
+        return getElement(elementLocator, locatorType);
+    }
+
+    protected String getElement(String elementLocator, LocatorStrategy locatorType) {
+        logger.info("Lookup element {} type {}", elementLocator, locatorType.name());
+        String result = null;
+        switch (locatorType) {
+        case START_STOP:
+            result = getValue(Locator.createLocatorByStartStop(elementLocator));
+            break;
+        case WIDTH:
+            result = getValue(Locator.createLocatorByWidth(elementLocator));
+            break;
+        }
+        return result;
+    }
+
+    private String getValue(Locator locator) {
+        Result result = connection.doCommand("ascii");
+        List<String> dataLines = result.getDataLines();
+        StringBuffer sb = new StringBuffer();
+        // When only one row is available
+        if (locator.getStartRow() == locator.getEndRow()) {
```

Comment:
The if, else if, else if is hard to read. What happens here is that the locator types are distinguished. Would be nicer to have type checks here instead.